### ISSUE TITLE
Login System Drive 

### ISSUE THREAD

**Issue by VidarrKerr** [veracrypt/VeraCrypt#1357]
It used to be that when you encrypted a system drive, after BIOS post, you would get a blank screen. Then, you would enter your passphrase. Now, it prints to screen "enter password". You used to ALSO have the option to write something to print to screen. I used to have, "OPERATING SYSTEM NOT FOUND...", or something similar. That was ideal.

This, "enter password" is ridiculous, because right away, any unauthorized person would know your drive is encrypted (or at least locked). 

I realize there is the other option for plausible deniability, but that requires having another, encrypted/unencrypted, partition/drive with an OS installed, so they think you are showing them what it is you have, while the other remains hidden. I don't want a second OS. 

Having a Single OS on an Encrypted system drive without giving it away, right away, that your disc is encrypted is ideal. 

Thanks

**Comment by Jertzukka** [comment]
Did you try editing the DcsProp configuration through "Edit boot loader configuration" in System Encryption Settings?

**Comment by VidarrKerr** [comment]
No, I did not. That is probably it. It was so long ago I forgot how I did
it.

Thanks!
